renovate.json

```diff
@@ -1,4 +1,13 @@
 {
```

Comment:
Suggest moving this to `renovate.json5`, can drop the quotes on keys and supports comments if we need.